tests/dummy/app/components/cookbook/external-model-table.js

```diff
@@ -0,0 +1,97 @@
+// BEGIN-SNIPPET external-model-table
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import Table from 'ember-light-table';
+import { isEmpty } from '@ember/utils';
+import { A } from '@ember/array';
+import diffAttrs from 'ember-diff-attrs';
+
+export default Component.extend({
+  columns: computed(function() {
+    return [{
+      label: 'Avatar',
+      valuePath: 'avatar',
+      width: '60px',
+      sortable: false,
+      cellComponent: 'user-avatar'
+    }, {
+      label: 'First Name',
+      valuePath: 'firstName',
+      width: '150px'
+    }, {
+      label: 'Last Name',
+      valuePath: 'lastName',
+      width: '150px'
+    }, {
+      label: 'Address',
+      valuePath: 'address'
+    }, {
+      label: 'State',
+      valuePath: 'state'
+    }, {
+      label: 'Country',
+      valuePath: 'country'
+    }];
+  }),
+
+  canLoadMore: true,
+  enableSync: true,
+
+  meta: null,
+  table: null,
+  rows: A([]),
+
+  didReceiveAttrs: diffAttrs('model', function(changedAttrs, ...args) {
```

Comment:
Might be worth switching to [ember-did-change-attrs](https://github.com/workmanw/ember-did-change-attrs) at some point but didn't now since it's still a WIP.